include/openenclave/attestation/sgx/evidence.h

```diff
@@ -64,7 +64,19 @@ OE_EXTERNC_BEGIN
     }
 
 /**
- * SGX specific claim: for the report data embedded in the SGX quote.
+ * SGX specific claim: SGX Quote verification collateral.
+ */
+#define OE_CLAIM_SGX_TCB_INFO "tcb_info"
```

Comment:
```suggestion
#define OE_CLAIM_SGX_TCB_INFO "sgx_tcb_info"
```
Suggest adding `sgx_` prefix to all SGX-specific claim IDs.